extensions/interactions/EndExploration/directives/end-exploration-validation.service.spec.ts

```diff
@@ -81,14 +82,18 @@ describe('EndExplorationValidationService', () => {
         refresher_exploration_id: null,
         missing_prerequisite_skill_id: null
       }),
-      null,
+      [],
       null
     )];
   });
 
   it('should not have warnings for no answer groups or no default outcome',
     () => {
       var warnings = validatorService.getAllWarnings(
+        // This throws "Type 'null' is not assignable to type 'Outcome'
+        // ." We need to suppress this error because of the need to test
+        // validations if no default outcome is defined.
+        // @ts-ignore
```

Comment:
If null is an accepted value, then add it to the function definition in the service. `defaultOutcome: Outcome | null)`

Reply:
done.